src/Symfony/Component/Notifier/Exception/UnsupportedOptionsException.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Exception;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Raphaël Geffroy <raphael@geffroy.dev>
+ */
+class UnsupportedOptionsException extends LogicException
+{
+    public function __construct(string $transport, string $supported, MessageOptionsInterface $given)
+    {
+        parent::__construct(\sprintf(
```

Comment:
please keep the call on one line

Reply:
Sorry, did not understand that the first time. Done ✅